langspec/xproc11/xproc.xml

```diff
@@ -1196,7 +1258,7 @@ readable port then the context node is undefined.</para>
               </listitem>
             </varlistentry>
             <varlistentry>
-              <term>Base URI</term>
+              <term>Static base URI</term>
               <listitem>
                 <para>The base URI of the element on which the expression occurs.</para>
```

Comment:
I used the same term as in XPath 3.1 specs. They say "Static Base URI", but of course we are free to choose an other term.